### ISSUE TITLE
fix JavaScript in Studio view

### ISSUE THREAD

**Issue by pdpinch** [mitodl/rapid-response-xblock#17]

**Comment by pdpinch** [comment]
@noisecapella is this still an issue?

**Comment by noisecapella** [comment]
No, @gsidebo fixed in in edx-platform: https://github.com/mitodl/edx-platform/pull/79/files#r169695486

**Comment by pdpinch** [comment]
I think we should open this upstream. It probably won't be merged before Hawthorn is cut, but it will still be easier to maintain if it's merged into master. 

@gsidebo let me know if you have any concerns about merging this upstream.

**Comment by gsidebo** [comment]
#24 covers the work to get these various XBlockAside changes merged upstream. Those changes include the Javascript fix. Closing this
